app/controls.go

```diff
@@ -14,9 +14,14 @@ import (
 
 // RegisterControlRoutes registers the various control routes with a http mux.
 func RegisterControlRoutes(router *mux.Router, cr ControlRouter) {
-	router.Methods("GET").Path("/api/control/ws").
+	router.
+		Methods("GET").
+		Path("/api/control/ws").
 		HandlerFunc(requestContextDecorator(handleProbeWS(cr)))
```

Comment:
It has no dynamic segments to the path, so the instrumentation will infer it automatically.